frontend/src/org/components/EventForm/steps/registration/ShowApplicationModal.tsx

```diff
@@ -200,103 +190,106 @@ export const ShowApplicationModal: FC<IShowApplicationModalProps> = ({
-            </div>
-          )}
-          {currentMemberships && currentMemberships.length !== 0 && (
-            <div>
-              <span>Členství: </span>
+              </div>
+            )}
+            {user.health_issues && (
               <div>
-                {currentMemberships.map(membership => {
-                  return (
-                    <Fragment
-                      key={`${membership.year}-${membership.administration_unit}-${membership.category}`}
-                    >
-                      {administrationUnits && categories && (
-                        <div>
-                          {
-                            administrationUnits.find(
-                              unit =>
-                                membership.administration_unit === unit.id,
-                            )?.name
-                          }
-                          {' - '}
-                          {
-                            categories.find(
-                              category =>
-                                membership.category.id === category.id,
-                            )?.name
-                          }{' '}
-                          ({membership.year})
-                        </div>
-                      )}
-                    </Fragment>
-                  )
-                })}
+                <span>Zdravotní omezení: </span>
+                <span>{user.health_issues}</span>
               </div>
-            </div>
-          )}
-        </div>
-      )}
-      {userId &&
+            )}
+            {user.close_person && (
+              <div>
+                <span>Blízká osoba: </span>
+                <span>{`${user.close_person.first_name} ${user.close_person.last_name}`}</span>
+                {user.close_person.email && (
+                  <span>
+                    , email:{' '}
+                    <EmailButton>{user.close_person.email}</EmailButton>
+                  </span>
+                )}
+                {user.close_person.phone && (
+                  <span>
+                    , tel: <PhoneButton>{user.close_person.phone}</PhoneButton>
+                  </span>
+                )}
+              </div>
+            )}
+            {currentMemberships && currentMemberships.length !== 0 && (
+              <div>
+                <span>Členství: </span>
+                <div>
+                  {currentMemberships.map(membership => {
+                    return (
+                      <Fragment
+                        key={`${membership.year}-${membership.administration_unit}-${membership.category}`}
+                      >
+                        {administrationUnits && categories && (
+                          <div>
+                            {
+                              administrationUnits.find(
+                                unit =>
+                                  membership.administration_unit === unit.id,
+                              )?.name
```

Comment:
Co se stane, pokud se nepodaří administrační jednotku najít? Vzhledem k tomu, že v query se načítají všechny administrační jednotky, spíše se to nestane, leda že by tam byla nějaká historická. Uvažuju, jestli tam nedat nějaký fallback, ale asi to není nutné.

Podobně níže s kategoriemi.